### DISCUSSION TITLE
Disable lsp but not highlighting for certain file types?

### DISCUSSION THREAD

**Discussion by seanbone** [helix-editor/helix#8137] [Q&A]
I'm writing CUDA code, and for whatever reason clangd has trouble with my *.cu files.

Is it possible to disable the LSP for *.cu files, but keep C++-style syntax highlighting?

If I remove *.cu from C++'s `file-types` entry in `languages.toml`, I lose syntax highlighting too. I tried adding the following to no avail:
```
[[language]]
name = "cuda"
scope = "source.cu"
injection-regex = "cpp"
file-types = ["cu", "cuh"]
auto-format = true
roots = []
comment-token = "//"
# language-servers = [ "clangd" ]
indent = { tab-width = 2, unit = "  " }
grammar = "cpp"
```

**Comment by pascalkuthe** [comment]
configs are merged automatically so:

``` toml
[[language]]
name = "cuda"
language-servers = []
```

you can also do `:lsp-stop` from within the editor

**Reply by seanbone** [reply]
The issue is that cuda isn't its own language in the default helix conf; the *.cu filetype is classed as C++. Adding just this complains about "missing field `scope` in `language`".
